### PR TITLE
add instructions for testing embedding

### PR DESCRIPTION
**Pull request by igraziatto**
I got these instructions with @nervoustwit earlier today. Thought it would be a good idea to update the docs with those as it might help someone in the future.

### CODE REVIEW

File: content/help/embedding/index.md [hunk 1]
```diff
@@ -56,6 +56,100 @@ Readers can log in using a Disqus ID, Twitter, Google, FaceBook or via single si
 
 The advanced embedding technique is for developers who require programmatic interaction between the Vanilla iframe and their parent window. It employs [easyXDM](http://easyxdm.net/wp/) and a special container layer to achieve this. It requires a more in-depth setup than Vanilla's basic embed solution.
 
+## Testing embedding locally on Docker
+
+If your local environment is set up with Docker you can follow the steps below to test embedding:
```
Comment (slafleche) [inline content/help/embedding/index.md]: Add a link to the vanilla-docker repo

File: content/help/embedding/index.md [hunk 2]
```diff
@@ -56,6 +56,100 @@ Readers can log in using a Disqus ID, Twitter, Google, FaceBook or via single si
...
+
+1. Create a “embed” directory in your workspace.
```
Comment (slafleche) [inline content/help/embedding/index.md]: Be a bit more explicit here, this is a little vague.

**Comment by Charrondev** [comment]
I think we should hold this document until we get the configs directly into vanilla-docker https://github.com/vanilla/vanilla-docker/issues/18.

Once that's done this whole could probably just be paragraph.

**Comment by charrondev** [comment]
I've updated this to use the `stub-embed-providers`.

**Pull request merged**